src/components/MintingCountdown.tsx

```diff
@@ -0,0 +1,18 @@
+import { Box } from "@mui/material"
+
+interface Props {
+  auctionStartFormatted: any,
+  auctionStartCountdown: any
+}
+
+const MintingCountdown = ({auctionStartFormatted, auctionStartCountdown}: Props) => {  
+  return (
+    <>
```

Comment:
I don't think you need to wrap this in a fragment since box is one top level component